applications/tari_base_node/src/commands/command/status.rs

```diff
@@ -94,7 +94,8 @@ impl CommandContext {
         );
 
         let conns = self.connectivity.get_active_connections().await?;
```

Comment:
You can do this using 
```rust
let conns = self.connectivity.select_connections(ConnectivitySelection::all_nodes(vec![])).await?;
```
without having to filter it yourself

Reply:
totally right, I missed that method. I just updated the PR.